src/custom/pages/Profile/index.tsx

```diff
@@ -63,8 +73,46 @@ export default function Profile() {
   const vCowBalanceUnvested = formatSmartLocaleAware(unvested, AMOUNT_PRECISION) || '0'
   const vCowBalance = formatSmartLocaleAware(total, AMOUNT_PRECISION) || '0'
 
-  const hasCowBalance = !cow?.equalTo(0)
-  const hasVCowBalance = !total?.equalTo(0)
+  const hasCowBalance = cow && !cow?.equalTo(0)
+  const hasVCowBalance = total && !total?.equalTo(0)
+  const hasVestedBalance = vested && !vested?.equalTo(0)
```

Comment:
I don't think you need the extra check, do you?

```suggestion
  const hasCowBalance = !cow?.equalTo(0)
  const hasVCowBalance = !total?.equalTo(0)
  const hasVestedBalance = !vested?.equalTo(0)
```

Reply:
Nah he does, double negative checks here. I thought the same but he confirmed with a jsfiddle

Reply:
This doesn't work https://jsfiddle.net/pksz7fy6/